src/common/charts/shared.js

```diff
@@ -0,0 +1,99 @@
+    lineChartSecondaryColor = 'var(--color-data-viz-line-chart-secondary)',
+    lineChartTertiaryColor = 'var(--color-data-viz-line-chart-tertiary)',
+    lineChartQueternaryColor = 'var(--color-data-viz-line-chart-queternary)',
+    lineChartQuinaryColor = 'var(--color-data-viz-line-chart-quinary)',
+    trendPositiveColor = 'var(--color-data-viz-trend-positive)',
+    trendNegativeColor = 'var(--color-data-viz-trend-negative)',
+    chartPrimaryColor = 'var(--color-data-viz-chart-primary)',
+    chartSecondaryColor = 'var(--color-data-viz-chart-secondary)',
+    chartTertiaryBackgroundColor = 'var(--color-data-viz-chart-tertiary-background)',
+    chartTertiaryStrokeColor = 'var(--color-data-viz-chart-tertiary-stroke)',
+    chartQuaternaryBackgroundColor = 'var(--color-data-viz-chart-quaternary-background)',
+    chartQuaternaryStrokeColor = 'var(--color-data-viz-chart-quaternary-stroke)',
+    chartQuinaryBackgroundColor = 'var(--color-data-viz-chart-quinary-background)',
+    chartQuinaryStrokeColor = 'var(--color-data-viz-chart-quinary-stroke)',
+    // patterns are in highcharts PatternOptionsObject format
+    // refer to https://api.highcharts.com/class-reference/Highcharts.PatternOptionsObject
+    pattern1 = {
+        pattern: {
+            path: {
+                // d is a standard SVG path definition string
+                // refer to https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/d
+                d: 'M0 0 L0 3', // draw a 3 until vertical line
+            },
+            width: 4.5, // defines the x bounds of the repeating pattern
+            height: 3, // defines the  y bounds of the repeating pattern
+            backgroundColor: chartTertiaryBackgroundColor, // sets the patterns background color
+            color: chartTertiaryStrokeColor, // sets the patterns stroke color
+            patternTransform: 'rotate(-60)', // rotates the path -60 degrees
+        },
+    },
+    pattern2 = {
+        pattern: {
+            path: {
+                // d is a standard SVG path definition string
+                // refer to https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/d
+                d: 'M0 0 L3 0',
+            },
+            width: 3, // defines the x bounds of the repeating pattern
+            height: 5, // defines the  y bounds of the repeating pattern
+            backgroundColor: chartQuaternaryBackgroundColor, // sets the patterns background color
+            color: chartQuaternaryStrokeColor, // sets the patterns stroke color
+        },
+    },
+    // configure the color sequences based on how many series are used
+    colorsSingle = [chartPrimaryColor],
+    colorsTwo = [chartSecondaryColor, chartPrimaryColor],
+    colorsThree = [chartSecondaryColor, chartPrimaryColor, pattern1],
+    colorsMore = [
+        chartSecondaryColor,
+        pattern1,
+        chartPrimaryColor,
+        pattern2,
+        chartQuinaryBackgroundColor,
+    ],
```

Comment:
@martin-ebay can I get some clarity here, so we can refactor this part.
So if we have 1 series, we use primary as the main one, correct?
So in two series we are setting the secondaryColor as the first one and primary as secondary? Or is it going backwards?
In three series (This is where I get extremely confused), we have the first one as Secondary, second one as primary and the third as pattern1 (whatever pattern 1 is)... 
And then in more than 3, we have secondary, pattern1 (pattern1 moves to before primary?), primary, pattern2, and then quinary?

What is going on here?!

Reply:
so this all can be answered by looking at the figma 
https://www.figma.com/file/DVhjKXFZoTna19ffUkBk34/Evo-Data-Viz?node-id=5%3A27013&t=NsZjMIAAC8ri6lAn-0

if you look at the colors for charts portion of the figma you will see a table with the five "colors" with two of them being patterns.

Then if you look at the examples of the stacked bar charts the colors used are different depending on the number of series of data.

I'm going to post a couple images in slack to help point you at the right part of the figma

Reply:
The more I think about it, the more this is off...
So if we have 1 series, the color is primary. 
If we have 2 series the first color is secondary, and then switches to primary.
If we have 3 series, then it goes primary, secondary, and then pattern1
If we have 4, then it goes to secondary, pattern1, primary, and then pattern2.

So the point of this was to make sure the order of lines look okay right? And this was called out as for accessibility so that the colors won't overlap or will be ordered correctly.... But what is to guarantee that series 1 will be the first one? 
What if series 1 has the largest amount of data on the y axis? Then the order would no longer look correct right and it would be on the top? I don't think you can guarantee that series 1 will always be the smallest dataset. It could overlap with others? 
Of course, on Figma everything looks nice and neat, but with real data, that no longer is the case. Things change, trends can overlap and then the charts will no longer look nice and neat like designers want them to. We need to figure out what the best way to overcome that would be, and I don't think by switching the series lines we would achieve that.